src/main.rs

```diff
@@ -190,6 +187,10 @@ struct Args {
     /// Whether to run the container in privileged mode.
     #[clap(long)]
     privileged: bool,
+
+    /// File to load examples from.
+    #[clap(long, default_value = "Examples.toml")]
```

Comment:
Rather than having a default to a path, use the `Examples.toml` form this repo when no path is set